tests/components/webostv/test_notify.py

```diff
@@ -6,12 +6,13 @@
 
 from homeassistant.components.notify import ATTR_MESSAGE, DOMAIN as NOTIFY_DOMAIN
 from homeassistant.components.webostv import DOMAIN
-from homeassistant.const import CONF_ICON, CONF_SERVICE_DATA
+from homeassistant.const import ATTR_ICON
 from homeassistant.setup import async_setup_component
 
 from . import setup_webostv
 from .const import TV_NAME
 
+ATTR_SERVICE_DATA = "data"
```

Comment:
I think it would be better to use the const from notify platform:
`from homeassistant.components.notify import ATTR_DATA` 
Since we use the same in our notify platform

Reply:
Yes, I forgot that we already have it there.